dotCMS/src/main/java/com/dotcms/tika/TikaUtils.java

```diff
@@ -384,6 +387,11 @@ public Map<String, Object> getForcedMetaDataMap(final File binFile,
             //Creating the meta data map to use by our content
             metaMap.putAll(this.buildMetaDataMap());
             metaMap.put(FileAssetAPI.CONTENT_FIELD, content);
+
+            //From Tika 2.x the title is mapped to dcTitle (https://cwiki.apache.org/confluence/display/TIKA/Migrating+to+Tika+2.0.0)
+            if(!metaMap.containsKey(TITLE_META_KEY.key()) && metaMap.containsKey(DC_TITLE_META_KEY.key())){
+                metaMap.put(TITLE_META_KEY.key(), metaMap.get(DC_TITLE_META_KEY.key()));
```

Comment:
Maybe a one liner like 
`metaMap.putIfAbsent(TITLE_META_KEY.key(),metaMap.get(DC_TITLE_META_KEY.key())`

Reply:
may be something like this 
```
metaMap.put(TITLE_META_KEY.key(), metaMap.getOrDefault(DC_TITLE_META_KEY.key(), "unknown"));
```